kas.yml

```diff
@@ -50,6 +50,11 @@ local_conf_header:
     SDKMACHINE = "x86_64"
     USER_CLASSES = "buildstats image-mklibs image-prelink"
     PATCHRESOLVE = "noop"
+    SSTATE_MIRRORS ?= "file://.* http://${LOCAL_PREMIRROR_SERVER}/${PROJECT_NAME}/sstate-cache/PATH"
```

Comment:
@Silentsky0 meta-rte is a public repository, someone who do not have access to our internal network will not be able to use the cache so it is better to remove those changes

Reply:
Removed all lines in `kas.yml` addressing internal cached tasks